README.md

```diff
@@ -73,13 +73,6 @@ Using yarn, run:
 yarn install
 ```
 
-> The site is built using Docusaurus: you may need to install Docusaurus before running the Wiki
-> locally.
-
-```bash
-yarn upgrade @docusaurus/core@latest @docusaurus/preset-classic@latest
```

Comment:
You shouldn't need to run this as long as the previous `yarn install` command was run as outlined.